config/config.js

```diff
@@ -4,6 +4,19 @@ var token = require("./slack_token.js").token;
 var private_key = require("./service_account_key.js").key;
 
 var config = {
+    // Unfortunately this all has to be at the top level due to sequelize-cli
```

Comment:
sequelize-cli is the command line tool we use to apply and generate migrations. It looks for it at the top level in the config.  :( 

I could, however, move it to a config/database.js or some such and have sequelize-cli use it from there, and import it into the db namespace in here.